pages/fpki_crawler_howto

```diff
@@ -0,0 +1,111 @@
+
+## Federal PKI Certificate Policy Validation
+For each validating certificate chain, each of the Federal PKI policy Object Identifiers (OIDs) will be validated for that chain.
+
+#### Federal PKI Policy OIDs
+| Federal PKI Policy | Policy OID                | Description                          |
+|--------------------|---------------------------|--------------------------------------|
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.1  | id-fpki-certpcy-rudimentaryAssurance |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.2  | id-fpki-certpcy-basicAssurance       |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.3  | id-fpki-certpcy-mediumAssurance      |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.4  | id-fpki-certpcy-highAssurance        |
+| COMMON             | 2.16.840.1.101.3.2.1.3.6  | id-fpki-common-policy                |
+| COMMON             | 2.16.840.1.101.3.2.1.3.7  | id-fpki-common-hardware              |
+| COMMON             | 2.16.840.1.101.3.2.1.3.8  | id-fpki-common-devices               |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.12 | id-fpki-certpcy-mediumHardware       |
+| COMMON             | 2.16.840.1.101.3.2.1.3.13 | id-fpki-common-authentication        |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.14 | id-fpki-certpcy-medium-CBP           |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.15 | id-fpki-certpcy-mediumHW-CBP         |
+| COMMON             | 2.16.840.1.101.3.2.1.3.16 | id-fpki-common-high                  |
+| COMMON             | 2.16.840.1.101.3.2.1.3.17 | id-fpki-common-cardAuth              |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.18 | id-fpki-certpcy-pivi-hardware        |
+| Federal bridge     | 2.16.840.1.101.3.2.1.3.19 | id-fpki-certpcy-pivi-cardAuth        |
+| Federal Bridge     | 2.16.840.1.101.3.2.1.3.20 | id-fpki-certpcy-pivi-contentSigning  |
+| FPKI SHA-1    | 2.16.840.1.101.3.2.1.3.21 | id-fpki-SHA1-medium-CBP              |
+| FPKI  SHA-1   | 2.16.840.1.101.3.2.1.3.22 | id-fpki-SHA1-mediumHW-CBP            |
+| FPKI SHA-1    | 2.16.840.1.101.3.2.1.3.23 | id-fpki-SHA1-policy                  |
+| FPKI SHA-1    | 2.16.840.1.101.3.2.1.3.24 | id-fpki-SHA1-hardware                |
+| FPKI SHA-1     | 2.16.840.1.101.3.2.1.3.25 | id-fpki-SHA1-devices                 |
+| FPKI SHA-1 | 2.16.840.1.101.3.2.1.3.26 | id-fpki-SHA1-authentication          |
+| FPKI SHA-1 | 2.16.840.1.101.3.2.1.3.27 | id-fpki-SHA1-cardAuth                |
+
+### Certificate Grouping
+The CAs are categorized by type and agency or company using information from the CA certificate DN.  The type can be U.S. Government, State or Commercial.  For U.S. Government, the Agency is extracted from the DN.  For State Governments, the state name is extracted.  For commercial CAs, the company name is extracted.
+
+# Output Formats
```

Comment:
I would suggest moving these up to the top - and the "processing, and how the files are being built" info to the bottom. 

?? thoughts?